apps/main/src/lend/components/DetailsUser/components/UserInfoPnl.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components'
+
+import { formatNumber, FORMAT_OPTIONS } from '@ui/utils'
+import useStore from '@/lend/store/useStore'
+export const UserInfoPnl = ({ userActiveKey }: { userActiveKey: string }) => {
+  const resp = useStore((state) => state.user.loansDetailsMapper[userActiveKey])
+
+  const { details, error } = resp ?? {}
+
+  if (error) return '?'
+
+  return (
+    <Pnl pnl={details?.pnl?.percentage}>
+      {!details?.pnl?.percentage || !isFinite(+details.pnl.percentage)
+        ? '-'
+        : formatNumber(+details.pnl.percentage, { ...FORMAT_OPTIONS.PERCENT })}
+    </Pnl>
+  )
+}
+
+const Pnl = styled.span<{ pnl: string | undefined }>`
+  color: ${({ pnl }) =>
+    pnl === undefined || !isFinite(+pnl)
```

Comment:
`+undefined === NaN` and `isFinite(NaN)` returns `false`
```suggestion
    !isFinite(+pnl)
```